src/Loader/web/Loader.js

```diff
@@ -0,0 +1,19 @@
+import styled, { keyframes } from 'styled-components';
+
+const spin = keyframes`
+  from {transform: rotate(0deg);}
+  to {transform: rotate(360deg);}
+`;
+
+const Loader = styled.div`
+  display: inline-block;
+  animation: ${spin} 1s linear infinite;
+  width: ${(props) => props.theme.px(5)};
```

Comment:
`width` and `height` should be taken from `props` since every application can use loader at different places.